src/mood.ts

```diff
@@ -126,6 +128,19 @@ export function meatMood(urKels = false, embezzlers = false): Mood {
     }
   }
 
+  if (
+    myClass() === $class`Accordion Thief` &&
+    myLevel() >= 15 &&
+    !(
+      getWorkshed() === $item`warbear LP-ROM burner` ||
+      have($item`warbear LP-ROM burner` || get("questG04Nemesis") === "finished")
+    )
+  ) {
+    if (have($skill`The Ballad of Richie Thingfinder`))
+      useSkill($skill`The Ballad of Richie Thingfinder`);
+    if (have($skill`Chorale of Companionship`)) useSkill($skill`Chorale of Companionship`);
```

Comment:
Don't you want to use both of these for all remaining casts, rather than just one at a time? See the bird code for an example.